tools/IoTEdgeDevOps/VstsPipelineSync/db/dbo.UpsertVstsRelease.sql

```diff
@@ -20,6 +21,7 @@ AS
 	BEGIN
 		UPDATE dbo.VstsRelease
 		SET [Name] = @Name,
+			[SourceBranch] = @SourceBranch,
```

Comment:
nit: spacing.

Reply:
The spacing looks fine in my editor. Weird